modules/ROOT/pages/db/database-documentation.adoc

```diff
@@ -402,14 +390,14 @@ Deletes data in a database.
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  A *Time unit* that qualifies the #queryTimeout. |  SECONDS | 
-| Fetch Size a| Number |  Indicates how many rows to fetch from the database when rows are read from a ResultSet. This property is required when streaming is true; in that case a default value (10) is used. |  | 
+** DAYS |  A time unit that qualifies the *Query Timeout* field |  SECONDS | 
+| Fetch Size a| Number |  Indicates how many rows to fetch from the database when rows are read from a resultSet. This property is required when streaming is true; in that case a default value (10) is used. |  | 
 | Max Rows a| Number |  Sets the limit for the maximum number of rows that any ResultSet object generated by this message processor can contain for the given number. If the limit is exceeded, the excess rows are silently dropped. |  | 
-| SQL Query Text a| String |  The text of the SQL query to execute. |  | x
-| Parameter Types a| Array of <<ParameterType>> |  Allows to optionally specify the type of one or more of the parameters in the query. If provided, you're not required to reference all of the parameters, but you cannot reference a parameter that is not present in the input values. |  | 
-| Input Parameters a| Object |  A Map in which keys are the name of an input parameter to be set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix (E.g: where id = :myParamName)).  The map's values will contain the actual assignation for each parameter. |  | 
+| SQL Query Text a| String |  The text of the SQL query to execute |  | x
+| Parameter Types a| Array of <<ParameterType>> |  Allows to optionally specify the type of one or more of the parameters in the query. If provided, you're not even required to reference all of the parameters, but you cannot reference a parameter not present in the input values |  | 
+| Input Parameters a| Object |  A Map which keys are the name of an input parameter to be set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a semicolon prefix (for example, where `ID = :myParamName`).  The map's values contain the actual assignment for each parameter. |  | 
```

Comment:
"semicolon" is wrong and I fixed it last time I updated this reference (which was recently). It's a colon, not a semicolon. This was incorrect in multiple places. I did a S&R. Please change back to the correct colon.

Reply:
Fixed throughout